doc/code/api/pennylane_cirq.SimulatorDevice.rst

```diff
@@ -0,0 +1,93 @@
+SimulatorDevice
+===============
+
+.. currentmodule:: pennylane_cirq
+
+.. autoclass:: SimulatorDevice
+   :show-inheritance:
+
+   .. rubric:: Attributes Summary
+
+   .. autosummary::
+
+      ~SimulatorDevice.author
+      ~SimulatorDevice.circuit_hash
+      ~SimulatorDevice.name
+      ~SimulatorDevice.obs_queue
+      ~SimulatorDevice.observables
+      ~SimulatorDevice.op_queue
+      ~SimulatorDevice.operations
+      ~SimulatorDevice.parameters
+      ~SimulatorDevice.pennylane_requires
+      ~SimulatorDevice.short_name
+      ~SimulatorDevice.shots
+      ~SimulatorDevice.state
+      ~SimulatorDevice.version
+
+   .. rubric:: Methods Summary
+
+   .. autosummary::
+
+      ~SimulatorDevice.active_wires
+      ~SimulatorDevice.apply
+      ~SimulatorDevice.capabilities
+      ~SimulatorDevice.check_validity
+      ~SimulatorDevice.execute
+      ~SimulatorDevice.execution_context
+      ~SimulatorDevice.expval
+      ~SimulatorDevice.generate_samples
+      ~SimulatorDevice.marginal_prob
+      ~SimulatorDevice.post_apply
+      ~SimulatorDevice.post_measure
+      ~SimulatorDevice.pre_apply
+      ~SimulatorDevice.pre_measure
```

Comment:
These are no longer being used since `QubitDevice`
```suggestion

```

Reply:
Oh, I believe the `api` folder is autogenerated by `automodapi`, and shouldn't be committed. It needs to be removed and added to the `.gitignore`.